### DISCUSSION TITLE
Stuck on a black screen after picking an image when running iOS app

### DISCUSSION THREAD

**Discussion by chiragkumaralgo** [capawesome-team/capacitor-plugins#357] [Q&A]
We are having a strange issue in our iOS app, we are running iOS app with ionic capacitor angular project.

We tried the @capawesome/capacitor-file-picker 
<ion-button expand="block" (click)="pickImages()" #profileImage>Pick Images</ion-button>
					  
In this when we select the image and click on Use Photo, we see a black screen, then we can not navigate to anywhere, one more thing we notice we are getting the image data in the click method (pickImages) but we are stuck on a black screen, so we are not able to see the selected image on our page.

**Comment by robingenz** [comment]
Thank you for your request.
Please provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can debug the issue.
